web/docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
 
   web:
     build: .
-#    image: labbsr0x/whisper-example/web:latest
+    #    image: labbsr0x/whisper-example/web:latest
```

Comment:
why not use the image name?

Reply:
@werberson actually, there isn't an image uploaded. I will erase this line.